README.md

```diff
@@ -206,7 +206,7 @@ type MongoServiceOptions = {
   type: 'MONGO';
   serviceName: string;
   username: pulumi.Input<string>;
-  password: pulumi.Input<string>;
+  password?: pulumi.Input<string>;
```

Comment:
Please also describe in the readme that if the password is not provided it will be auto-generated.